lib/puppet/file_system/windows.rb

```diff
@@ -134,10 +137,30 @@ def replace_file(path, mode = nil)
              dacl
            when 0640, 0600
              secure_dacl(current_sid)
+           when 0660
+             dacl = secure_dacl(current_sid, FILE_RW, FILE_RW)
+             dacl
```

Comment:
is `secure_dacl(current_sid, FILE_RW, FILE_RW)` enough?

Reply:
660 is read and write for user and group, so this should be enough